playbook/app/pb_kits/playbook/pb_file_upload/file_upload.html.erb

```diff
@@ -5,11 +5,17 @@
     id: object.id,
     **combined_html_options) do %>
       <%= pb_rails("form_group", props: {cursor: "pointer", full_width: object.full_width}) do %>
-        <label for="upload-<%= object.id %>" class="pb_button_kit_secondary_inline_enabled"><%= "#{object.label}" %></label>
+        <label
+          for="upload-<%= object.id %>"
+          class="pb_button_kit_secondary_inline_enabled <%= 'dark' if object.dark %>"
```

Comment:
Can you move this class name to a method in `playbook/app/pb_kits/playbook/pb_file_upload/file_upload.rb`?

And also, make it so there's no hanging space at the end of the string?